### ISSUE TITLE
Add Ceph dashboard

### ISSUE THREAD

**Issue by mnaser** [vexxhost/atmosphere#408]
The Ceph dashboard is a useful tool in order to view the details of the Ceph cluster, it would be beneficial to enable it to be exposed to users via an ingress.

**Comment by thywyn** [comment]
You all planning on implementing the dashboard via Rook?

**Comment by lukasmrtvy** [comment]
There is still an issue with loop redirect when a user does not exist in Ceph, see for more info https://tracker.ceph.com/issues/62034

**Comment by mnaser** [comment]
> You all planning on implementing the dashboard via Rook?

Good idea. If we take this route it will help us speed the process up.

I think the dashboard runs inside the manager though so it has to live through it.

**Comment by mnaser** [comment]
> There is still an issue with loop redirect when a user does not exist in Ceph, see for more info https://tracker.ceph.com/issues/62034

I'm thinking we can play around this with some sort of proxy... 🤔
